transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -370,6 +370,10 @@ static int init_in_addr(JNIEnv* env, jbyteArray address, struct in_addr* addr) {
 }
 // util methods end
 
+// prototypes for the load hook from DirectFileDescriptorController.c
+void directFile_JNI_OnLoad(JNIEnv* env);
```

Comment:
we should return `jint` here so we can return JNI_ERR if something bad happens. WDYT ?